js/binary/utils_test.js

```diff
@@ -715,6 +715,7 @@ describe('binaryUtilsTest', function() {
     var sourceBuffer = sourceBytes.buffer;
     var sourceBase64 = goog.crypt.base64.encodeByteArray(sourceData);
     var sourceString = goog.crypt.byteArrayToString(sourceData);
+    var sourceUint8Array = new Uint8Array(sourceData)
```

Comment:
nit: Is this actually testing something new? We already test `sourceBytes` which looks to be identical (in other words: I assume this test change would pass even without the implementation change above).

It should be enough to use a subclass here, e.g. `new (class extends Uint8Array {})(sourceData)`.